libs/sdk-ui-ext/src/internal/translations/en-US.json

```diff
@@ -781,6 +781,11 @@
         "comment": "",
         "limit": 0
     },
+    "properties.total.measures.tooltip": {
+        "value": "Disable “is total” options in metric to add sum of all values as a column at the end.",
```

Comment:
Please check with @NestorEncinas if these special quotes are ok for the translations export/import.

Reply:
It should be fine, we already have some texts with same format.